lib/tmdb_handler.rb

```diff
@@ -198,87 +198,4 @@ def tmdb_handler_search_common_actors_in_two_movies(movie_one_title, movie_two_t
       )
     end
   end
-
-  def tmdb_handler_discover_search(params)
```

Comment:
This has been replaced by the new `Tmdb::Client.movie_advanced_search(params)` method